hermes-plugin/src/main/groovy/com/binxhealth/hermes/message/MessageCommand.groovy

```diff
@@ -19,7 +19,7 @@ class MessageCommand implements Validateable {
     def metadata
 
     static constraints = {
-        url nullable: false, url: ['localhost:\\d*']
+        url nullable: false, url: ['localhost:\\d*', '\\S+.local']
```

Comment:
Whoops - actually should this be \\s+ rather than \\S+ ? Or perhaps it doesn't matter?